decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './nameable'
+
+class Decorator < Nameable
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable
+  end
```

Comment:
- [ ] The correct_name method for the `Decorator` class should return the `correct_name` method of the `nameable` object it takes in as a parameter during initialization. Kindly implement this change. 👍